.ci/setup_env_ubuntu.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -e
+#
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cidir=$(dirname "$0")
+source "/etc/os-release"
+source "${cidir}/lib.sh"
+get_cc_versions
+
+if grep -q "N" /sys/module/kvm_intel/parameters/nested; then
```

Comment:
This is `x86_64`-specific, so could you add something like the following around this block:

```
source "${cidir}/lib.sh"

arch=$(arch)

if [ "$arch" = x86_64 ]; then
    ...
else
   die "Unsupported architecture: $arch"
fi
```

Note that we'll need to define `die` in `lib.sh` too.